lighthouse-core/fraggle-rock/gather/navigation-runner.js

```diff
@@ -287,7 +287,8 @@ async function navigation(options) {
     skipAboutBlank: configContext.skipAboutBlank,
   };
 
-  return Runner.run(
+  const runner = new Runner(config, computedCache);
```

Comment:
IMO if it's just a glorified options object it should just be an options object. There's no complicated state being shared or encapsulated, it's just a cache and an immutable config, so no reason to have a constructed object.

Ideally these functions would be as independent as possible (possibly eventually in separate files) to avoid ease of unseparating concerns in the future

Reply:
Aight I'll back down from this